qds_sdk/account.py

```diff
@@ -94,7 +94,7 @@ def branding(args):
         return result
 
 class Account(SingletonResource):
-    credentials_rest_entity_path = "accounts/get_creds"
+    credentials_rest_entity_path = "accounts/creds"
```

Comment:
What is the motivation for this change ? 
It this change is irrelevant to this PR, then lets take it up in a separate PR.